data/volunteersMap.json

```diff
@@ -1,26 +1,27 @@
 {
-    "0x1fB8CD37C35546FeC07A794158F4035f06f023A0": "Loredana",
-    "0x0092D68117A00c6D8CC651a38522AAEEd54ba3F2": "Christian",
-    "0xFDb7Af458A29bF7bF9CDa9d4cba2149F3217763c": "Feynman",
-    "0x4eDf957557F28AFeb54B3722dB09737769e26cbE": "Claudiu",
-    "0xfa28ED428D54424D42ED4F71415315df2f2E49D6": "MaxxFlyer",
-    "0x8AFb5ec02ac1E57c5e8d3a5422D73809d4630f1D": "Vedlerual",
-    "0x2C4aC604173611b26fC3e8A4a82c1207Be467238": "VGrazzini",
-    "0x6cAbB70a0Bfe74EC841110e781432ad0f38D0b89": "RoOddball",
-    "0x3656BcC28140536a9614d1d7f7Dc4ddf4dE0E373": "JRoquette",
-    "0xf1571B33be790965340E82f28a9cEE629260F642": "hemantech",
-    "0xbA121F230078c658dC3f70441C7E12406B9A1Ee3": "Moe",
-    "0x4213091AD181560C69F68A3a55530889CE8B052d": "ghassanmas",
-    "0x6E1DDf485c8d73db8751D867067BbF3043eA98b4": "incredulous",
-    "0x20aBa8cDCF10Dd5e5645d684EC99B821bC9d5C89": "IceDragon",
-    "0xD2170bd6477EB86Cb23237172764Ff3b20254Cfd": "Soumik",
-    "0x7Bbe29411c6B55b0EcFea4efaEEc6dA3fbDB971D": "Jimmy",
-    "0x46805B81DeBB9606c81e5C1dD74589e9BF92778b": "OneEyeDoll",
-    "0x9FDf825f11E9529e84C202A61982Ed4F3E208cde": "caboom",
-    "0x01494446B60D3193bE928AB888Fe702d0E903B3d": "MoAnas",
-    "0xD95Fa8720d52f8c8F45C8782138Eb4d4DdE10111": "Gracey",
-    "0x3354723cA68106B121f6FfABF1aFE4BCA6CBe2Be": "Leo",
-    "0x925207D5Eff0eE8a330Cb88213751256A2E4E50A": "SuyashMane",
-    "0xE0E5fF9D8a39D7bE1FF0a240d3276385c72409DB":"bam",
-    "0x245cB7Fb7C3Aed28f48b4cd72A35B0114cA1D67e":"Hazem"
-}
+  "0x1fB8CD37C35546FeC07A794158F4035f06f023A0": "Loredana",
+  "0x0092D68117A00c6D8CC651a38522AAEEd54ba3F2": "Christian",
+  "0xFDb7Af458A29bF7bF9CDa9d4cba2149F3217763c": "Feynman",
+  "0x4eDf957557F28AFeb54B3722dB09737769e26cbE": "Claudiu",
+  "0xfa28ED428D54424D42ED4F71415315df2f2E49D6": "MaxxFlyer",
+  "0x8AFb5ec02ac1E57c5e8d3a5422D73809d4630f1D": "Vedlerual",
+  "0x2C4aC604173611b26fC3e8A4a82c1207Be467238": "VGrazzini",
+  "0x6cAbB70a0Bfe74EC841110e781432ad0f38D0b89": "RoOddball",
+  "0x3656BcC28140536a9614d1d7f7Dc4ddf4dE0E373": "JRoquette",
+  "0xf1571B33be790965340E82f28a9cEE629260F642": "hemantech",
+  "0xbA121F230078c658dC3f70441C7E12406B9A1Ee3": "Moe",
+  "0x4213091AD181560C69F68A3a55530889CE8B052d": "ghassanmas",
+  "0x6E1DDf485c8d73db8751D867067BbF3043eA98b4": "incredulous",
+  "0x20aBa8cDCF10Dd5e5645d684EC99B821bC9d5C89": "IceDragon",
+  "0xD2170bd6477EB86Cb23237172764Ff3b20254Cfd": "Soumik",
+  "0x7Bbe29411c6B55b0EcFea4efaEEc6dA3fbDB971D": "Jimmy",
+  "0x46805B81DeBB9606c81e5C1dD74589e9BF92778b": "OneEyeDoll",
+  "0x9FDf825f11E9529e84C202A61982Ed4F3E208cde": "caboom",
+  "0x01494446B60D3193bE928AB888Fe702d0E903B3d": "MoAnas",
+  "0xD95Fa8720d52f8c8F45C8782138Eb4d4DdE10111": "Gracey",
+  "0x3354723cA68106B121f6FfABF1aFE4BCA6CBe2Be": "Leo",
+  "0x925207D5Eff0eE8a330Cb88213751256A2E4E50A": "SuyashMane",
+  "0xE0E5fF9D8a39D7bE1FF0a240d3276385c72409DB": "bam",
+  "0x245cB7Fb7C3Aed28f48b4cd72A35B0114cA1D67e":"Hazem"
```

Comment:
missing comma at the end of row 25.

Reply:
Done

Reply:
Not done. The comma is on the wrong line. It should be on row 25, not 26.